pom.xml

```diff
@@ -63,6 +63,7 @@
         <version.jackson>2.10.2</version.jackson>
         <version.org.slf4j>1.7.30</version.org.slf4j>
         <version.log4j>1.2.17</version.log4j>
+        <version.netty>4.1.50.Final</version.netty>
```

Comment:
Shouldn't we then put a definition for netty into the `<dependencyManagement>` section?

Reply:
Good point indeed.

Reply:
No, given the increasing amount of alignments I'd keep it as is and move https://issues.redhat.com/browse/DBZ-2145 to `1.5`

Reply:
Isn't there are need for more alignment because there's no major `<dependencyManagement>` definiton in that debezium-parent pom yet? In-fact that section is already our BOM?!